### DISCUSSION TITLE
Searching through original/origin path(s)

### DISCUSSION THREAD

**Discussion by Snify89** [photoprism/photoprism#3602] [Ideas]
Right now it seems that you can only search original filenames but not the path.

e.g.
original:"IMG_9831-112"

https://docs.photoprism.app/user-guide/search/filters/

It would be nice to also search (sub) folder names.

Something like that:
original_path:"myalbumfolder"
or
originalpath:"myalbumfolder"

Thank for your considering :)

**Comment by graciousgrey** [comment]
This is already possible by using  using wildcards :)

You can search for:

 `original:"myalbumfolder*"` 
`original:"*myablumfolder*"`
`original:"myalbumfolder/IMG123"`

**Comment by loulou91** [comment]
I made a test, it doesn't work with "original:" but is fine with "folder:"
;-)

**Reply by graciousgrey** [reply]
The path and folder filters look for the actual location of the file in the originals folder.
The original filter looks for the filename a file had before it has been imported. Files that have only been indexed and not imported don't have a original filename property as it is the same as the actual filename.
